### ISSUE TITLE
Errors occurring outside tests no longer reported correctly

### ISSUE THREAD

**Issue by unikitty37** [archfz/cypress-terminal-report#85]
Since upgrading to 3.0.0, error reporting seems to be broken for errors occurring outside tests. (This also applies to 3.0.1).

Given the following test file:

```js
describe('bug', () => {
  cy.log('not in a test')
})
```

Version 2.4.0 reports it correctly:

![image](https://user-images.githubusercontent.com/139183/111479298-bdf00f00-8728-11eb-9dd5-1da6b119771c.png)

But versions 3.0.0 and 3.0.1 hit an error in LogCollectSimpleControl.js:

![image](https://user-images.githubusercontent.com/139183/111479911-56868f00-8729-11eb-83e7-c1e32fe22968.png)

```
TypeError: Cannot read property 'relativeFile' of undefined

Because this error occurred during a `after each` hook we are skipping all of the remaining tests.
    at LogCollectSimpleControl.sendLogsToPrinter (LogCollectSimpleControl.js:25)
    at Context.eval (LogCollectSimpleControl.js:79)
logError @ cypress_runner.js:197884
```

**Comment by archfz** [comment]
On a note I believe executing command of cypress on that level is not correct right?

**Comment by unikitty37** [comment]
Right — but the Cypress error message is a lot more helpful for finding what you did wrong 😁

**Comment by archfz** [comment]
Released in 3.0.2

**Comment by jmoses** [comment]
Is a fix for this really released?  I looked in the changelog between 3.0.1 and 3.0.2 and the only addition was a test case that would express the error, but it's doesn't _look_ like it's being run from `test/test.js` anywhere as part of the suite.

**Comment by archfz** [comment]
Apparently I stashed at some point some of the changes. My mistake on that release. Fixed release in 3.0.3
